website/docs/security/authentication.md

```diff
@@ -6,46 +6,48 @@ description: Learn to use authentication mechanisms
 
 # Authentication
 
-FerretDB does not store authentication information (usernames and passwords) itself but uses the backend's authentication mechanisms.
-The default username and password can be specified in FerretDB's connection string,
-but the client could use a different user by providing a username and password in MongoDB URI.
-For example, if the server was started with `postgres://user1:pass1@postgres:5432/ferretdb`,
-anonymous clients will be authenticated as user1,
-but clients that use `mongodb://user2:pass2@ferretdb:27018/ferretdb?tls=true&authMechanism=PLAIN` MongoDB URI will be authenticated as user2.
-Since usernames and passwords are transferred in plain text,
+FerretDB supports `PLAIN`, `SCRAM-SHA-256` and `SCRAM-SHA-1` authentication mechanisms.
```

Comment:
Is this the correct file to update? I see there are versioned documents now and `task docs-dev` runs on versioned one https://github.com/FerretDB/FerretDB/blob/main/website/versioned_docs/version-v1.20/security/authentication.md instead of this document

Reply:
That's the correct file to edit.

`task docs-dev` renders all files. There is a version selection on the top right.

Reply:
Great thanks!